src/test/resources/org/mapfish/print/config/sample_config_yaml/configNonPremiumGoogle.yaml

```diff
@@ -44,5 +44,4 @@ layouts:
         - !map
           spacingAfter: 30
           width: 440
-          height: 483
-      footer: *commonFooter
```

Comment:
Why we removed the `footer` here?

Reply:
also a malformed yaml. The * character is [an anchor](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/), which is used to reference previously defined with & character property (for example &commonFooter), but in this file there are no such definition, which results in exception (for correct example of & and * usage you can see [configMultipleMaps.yaml](https://github.com/geosolutions-it/mapfish-print-v2/blob/d91e04ea89dde4015a8675de9648801d8727d623/src/test/resources/org/mapfish/print/config/sample_config_yaml/configMultipleMaps.yaml#L101)).